EpsilonDriverControls/Inc/DriverControls.h

```diff
@@ -36,7 +36,7 @@
 #define NON_ZERO_THRESHOLD 0.17f
 #define MAX_PEDAL_THRESHOLD 0.71f
 #define MAX_ANALOG 4095 // 12bit ADC (2^12)
-#define REGEN_INPUT_SCALING 0.5f
```

Comment:
This is still removed

Reply:
Oops, forgot to save the header file haha